src/Common/src/Interop/Windows/Interop.Errors.cs

```diff
@@ -83,5 +83,6 @@ internal partial class Errors
         internal const int ERROR_RESOURCE_LANG_NOT_FOUND = 0x717;
         internal const int EFail = unchecked((int)0x80004005);
         internal const int E_FILENOTFOUND = unchecked((int)0x80070002);
+        internal const int ERROR_EVENTLOG_FILE_CHANGED = 1503;
```

Comment:
Nit: To match the style of the rest of the file, this should be: 

`internal const int ERROR_EVENTLOG_FILE_CHANGED = 05DF;`

And it should be in the list between `ERROR_CLASS_ALREADY_EXISTS` and `ERROR_TRUSTED_RELATIONSHIP_FAILURE`.